english/pokemon/pikachu_wins.md

```diff
@@ -1,6 +1,6 @@
 You bring out your Pikachu and Pikachu uses thunderbolt on the marshmallow! You win!
 
-However, you don't get to enjoy your victory for long.  You awake from your dream and
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.